From 5a650b55d8375894c1269f0f82ebb86a47dd3df2 Mon Sep 17 00:00:00 2001
From: Jim Basney <jbasney@illinois.edu>
Date: Tue, 28 Oct 2014 14:49:12 -0400
Subject: [PATCH] in MyProxy.bootstrapTrust(), don't force SSLv3, so TLS is
 used by default and there are no compatibility problems with MyProxy servers
 that don't support SSLv3

---
 myproxy/src/main/java/org/globus/myproxy/MyProxy.java | 1 -
 1 file changed, 1 deletion(-)

diff --git a/myproxy/src/main/java/org/globus/myproxy/MyProxy.java b/myproxy/src/main/java/org/globus/myproxy/MyProxy.java
index 53b8431..148c000 100644
--- a/myproxy/src/main/java/org/globus/myproxy/MyProxy.java
+++ b/myproxy/src/main/java/org/globus/myproxy/MyProxy.java
@@ -1170,7 +1170,6 @@ public class MyProxy  {
             sc.init(null, trustAllCerts, new java.security.SecureRandom());
             SSLSocketFactory sf = sc.getSocketFactory();
             SSLSocket socket = (SSLSocket)sf.createSocket(this.host, this.port);
-            socket.setEnabledProtocols(new String[] { "SSLv3" });
             socket.startHandshake();
             socket.close();
 
-- 
2.4.3

